RTN-006.tex

```diff
@@ -50,6 +52,386 @@
+\item collecting and evaluating data about RSP users (and co-authors of Rubin-related journal articles) to understand which communities are accessing the LSST data products, and using this to inform further proactive DEI efforts
+\end{itemize}
+
+The CET acknowledges that DEI-focused strategies for engagement were also raised as issues that are very important to the science community during the CET's virtual open discussion series on community engagement in late 2020.
+Future versions of this community engagement model will feature more fully developed descriptions of these proactive, DEI-driven engagement strategies.
+
+
+\subsubsection{User Acceptance Testing of the RSP}\label{sssec:mod_dev_uat}
+
+The CET will conduct User Acceptance Testing (UAT) of the RSP and associated community-facing services at regular intervals, such as prior to major updates to the RSP, or before an annual Data Release.
+
+The components and workflow for the UAT process have yet to be defined, but might include:
+\begin{itemize}
+\item coordinating with the Users Committee (\S~\ref{sssec:mod_coord_uc}) to include RSP users from the community in UAT campaigns,
+\item reporting issues and coordinating with Rubin Observatory teams to resolve them, and
+\item producing an acceptance test report to the Rubin Observatory product owners.
+\end{itemize}
+
+
+\subsubsection{Citizen Science Methodologies}\label{sssec:mod_dev_citizen}
+
+Given the unique size and complexity of the LSST data set there will be certain scientific results that are only obtainable via citizen science methodologies.
+The CET will help scientists identify and prepare LSST data and documentation for citizen science programs, and will coordinate with the EPO department and Zooniverse to help ensure that the data are installed and the outputs will meet the program's science goals.
+
+The CET's activities related to citizen science are still in the early stages of development.
+
+
+
+
+
+
+\section{Proposed Operations Work Packages}\label{sec:comp}
+
+The following is a very brief summary of the proposed work packages and staffing profile for the CET, which have been generated in tandem with the development of Version 1 of this community engagement model.
+
+
+\subsection{Proposed Work Breakdown Structure}\label{ssec:comp_wbs}
+
+As of Version 1 of this document, these are the five proposed components of the CET's work breakdown structure (WBS) packages.
+
+\begin{enumerate}
+\item \textbf{Coordinate Support for Science:} curate the community forum, guide scientists to help each other find solutions, identify emergent issues, and coordinate investigations to resolve them.
+\item \textbf{RSP User Acceptance Testing (UAT):} Conduct UAT campaigns in coordination with the Users Committee prior to major updates, and pass along reports to the appropriate internal departments and follow-up on any issues.
```

Comment:
.... reports to the Rubin Observatory product owners and appropriate internal ....

Reply:
Full statement is now: "Conduct UAT campaigns in coordination with the Users Committee prior to major updates, report to the Rubin Observatory product owners and appropriate internal departments, and follow-up on any issues."